content/projects/infrastructure/index.adoc

```diff
@@ -52,25 +52,42 @@ we have a follow the sun policy where we try to only be on call during our "avai
 Another major aspect of the Jenkins Infrastructure project, is the maintenance of all services either provided by us or third parties.
 Here is a non-exhaustive list of services that we provide and maintain.
 
+=== Self-hosted services
+
+|===
+| Service                                                | JIRA component(s) | Repository
+| https://accounts.jenkins.io[Accounts]                  | `account`, `accountapp` | https://github.com/jenkins-infra/account-app[Code] 
+| link:./ircbot[IRC Bot]                                                | `ircbot` | https://github.com/jenkins-infra/ircbot[Code]
+| https://javadoc.jenkins.io[Javadoc]                    | `javadoc` | https://github.com/jenkins-infra/javadoc[Code]
+| https://issues.jenkins-ci.org[Jira]                    | `jira` | https://github.com/jenkins-infra/jira[Docker] 
+| https://ci.jenkins.io[Jenkins]                         | `ci.jenkins.io`, `ci` | https://github.com/jenkins-infra/jenkins-infra[Deployment scripts]
+| LDAP                          | `ldap` | https://github.com/jenkins-infra/ldap[Docker]
+| https://jenkins.io[Main Website]                       | `jenkins.io` | https://github.com/jenkins-infra/jenkins.io[Code]
+| VPN          | `vpn` | https://github.com/jenkins-infra/openvpn[Code]
+| https://plugins.jenkins.io[Plugin Site]                       | `plugins.jenkins.io` | https://github.com/jenkins-infra/plugin-site[Frontend], https://github.com/jenkins-infra/plugin-site-api[Backend]
+| http://stats.jenkins.io/jenkins-stats/svg/svgs.html[Stats] | N/A | https://github.com/jenkins-infra/infra-statistics[Scripts]
+| https://uplink.jenkins.io[UpLink]                | `uplink` | https://github.com/jenkins-infra/uplink[Code] 
+| https://wiki.jenkins.io[Wiki]                          | `wiki` | https://github.com/jenkins-infra/confluence[Docker]
+|===
+
+=== SaaS
+
+|===
+| Service                                                | Provided by  | JIRA component(s) | Repository
+| https://repo.jenkins-ci.org/webapp/#/home[Artifactory] | JFrog | `artifactory` | JFrog
```

Comment:
We don't identify GitHub as the "Repository" for the content they host.  I recommend we use the same technique with JFrog and act as though the repository they are using is opaque to us.  We can't see it, but we are very grateful they provide it.
```suggestion
| https://repo.jenkins-ci.org/webapp/#/home[Artifactory] | JFrog | `artifactory` | -
```